fahrschein-spring-boot-starter/src/main/java/org/zalando/spring/boot/fahrschein/config/Registry.java

```diff
@@ -110,6 +111,15 @@ public static <T> List<T> list(final T... elements) {
         return list;
     }
 
+    public <T> Map<String, T> getBeansOfType(Class<T> clazz) {
+        if (registry instanceof DefaultListableBeanFactory) {
+            DefaultListableBeanFactory factory = (DefaultListableBeanFactory) registry;
+            return factory.getBeansOfType(clazz);
+        } else {
+           LOG.warn("Unable to get beans of type {} from registry of type: {}", clazz.getName(), registry.getClass().getName())
```

Comment:
@RameshMalla actually, I would consider these errors (more drastically in `registerAliasesForNakadiListener` below: If we cannot pick up NakadiEventListener annotated beans, the whole application setup is broken..)